python/setup.py

```diff
@@ -54,7 +42,7 @@
     'scikit-learn',
     'packaging',
     'interpret-core[required]==0.1.19',
-    'shap>=0.20.0, <=0.32.1'
+    'shap>=0.20.0, <=0.33.0'
```

Comment:
Does this kind of change necessitate a bump to 0.4.0? I'd like to get more clarity on when we patch and when we bump minor version

Reply:
I always do a separate PR for the version bump to make it clear in the commit history